src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryPipelineSpark.java

```diff
@@ -186,7 +189,7 @@ protected void runTool( final JavaSparkContext ctx ) {
 
         // TODO: 1/14/18 this is the next version of precise variant calling
         if ( expInterpret != null ) {
-            experimentalInterpretation(ctx, assembledEvidenceResults, svDiscoveryInputMetaData);
+            experimentalInterpretation(ctx, expInterpret, assembledEvidenceResults, svDiscoveryInputMetaData);
```

Comment:
What's the use of this boolean variable? Why not just add the check that it's true to the if clause on the previous line, then you don't need the extra parameter.

Reply:
Right! I think the original CMD line option was a string telling where to store the whole directory of experimental interpretation, and was passed in as a String. When I removed the directory creation and switched to the flag, I forgot to remove this parameter.